okhttp-clients/src/test/java/com/palantir/remoting3/okhttp/OkHttpClientsTest.java

```diff
@@ -167,28 +135,10 @@ public void throwsProperRemoteExceptionAfterRetry() throws Exception {
                 .setResponseCode(400);
         server.enqueue(mockResponse);
 
-        OkHttpClient client = OkHttpClients.withCustomQosHandler(
-                createTestConfig(url),
-                AGENT,
-                OkHttpClientsTest.class,
-                () -> retryingHandler);
+        OkHttpClient client = createRetryingClient(5);
 
         Call call = client.newCall(new Request.Builder().url(url).build());
-        assertThatThrownBy(call::execute).isInstanceOf(RemoteException.class);
-    }
-
-    @Test
-    public void interceptsAndHandlesQosIoExceptions_wrapsRuntimeExceptionsAsIoExceptions() throws Exception {
```

Comment:
note: I removed this test, because we want to propagate `RemoteException` as-is and not wrap it. This means that normal `RuntimeException` are also propagated as-is, but I couldn't see any real issue with that.